BUILD.md

```diff
@@ -125,11 +125,12 @@ The TensorRT execution provider for ONNX Runtime is built and tested with CUDA 9
  - The path to the CUDA `bin` directory must be added to the PATH environment variable so that `nvcc` is found.
  - The path to the CUDNN installation (path to folder that contains libcudnn.so) must be provided via the CUDNN_PATH environment variable, or `--cudnn_home parameter`. 
 - The path to TensorRT installation must be provided via the `--tensorrt_home parameter`.
+- There are two kind of protobufs: protobuf and protobuf-lite. Currently TensorRT execution provider only supports protobuf by using `--use_full_protobuf` option.
```

Comment:
is full_protobuf the same as protobuf? What is the flag for protobuf-lite?

Reply:
Yes. If there is no use_full_protobuf, protobuf-lite will be used.

Reply:
At some point you should plan to move to protobuf-lite due to the significant saving of binary size.

Reply:
Yes. This will be an action item for onnx-tensorrt team

Reply:
why not make this a default flag instead of asking the use to include.

Reply:
because protobuf-lite now is the default setting for onnxruntime. protobuf-lite hasn't been supported by trt parser yet